core/pb/xchain.proto

```diff
@@ -1022,3 +1034,19 @@ message ContractStatData {
   int64 accountCount = 1;
   int64 contractCount = 2;
 }
+
+// Query address contracts request
+message AddressContractsRequest {
+  Header header = 1;
+  string bcname = 2;
+  string address = 3;
+  bool need_content = 4;
```

Comment:
What's the content meaning?

Reply:
Only contract name, or detailed contract info like txid,timestamp,isbanned etc.